cli/crates/cli/Cargo.toml

```diff
@@ -38,10 +38,10 @@ duct = "0.13.5"
 reqwest = { version = "0.11.11", features = ["json", "blocking"] }
 serde = { version = "1", features = ["derive"] }
 serde_json = "1.0.85"
-sysinfo = "0.25.3"
+sysinfo = "0.26.2"
```

Comment:
@yoav-lavi should we adopt the convention we use in the `api` repo, which is to specify only the minor version for `0.x` versions and only major such as `1`, `2` etc.?

Reply:
IIRC we did, this may have been added after, will look into it

Reply:
https://github.com/grafbase/grafbase/pull/78